domains/gravy.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "thatgravyboat",
+    "twitter": "thatgravyboat",
+    "discord": "thatgravyboat#0001"
```

Comment:
email pls? if not wanna leak, leave an empty string like `{"email": ""}`

Reply:
Fixed